src/scripts/main.js

```diff
@@ -1,3 +1,66 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('table tbody');
+
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+const max = 10;
+const min = 2;
+
+const setButton = (el, state) => (el.disabled = state);
+
+appendRow.addEventListener('click', (e) => {
+  const rows = [...tbody.querySelectorAll('tr')];
+
+  rows.push(rows[1].cloneNode(true));
+  rows.forEach((row) => tbody.appendChild(row));
+
+  if ([...tbody.querySelectorAll('tr')].length === max) {
+    setButton(e.target, true);
+  } else {
+    setButton(removeRow, false);
+  }
+});
+
+removeRow.addEventListener('click', (e) => {
+  tbody.removeChild(tbody.lastElementChild);
```

Comment:
Here, you are removing the last child of the table body without checking if it exists. This could lead to an error if there are no more rows to remove. Always check if the element exists before trying to remove it.